README.md

```diff
@@ -2,56 +2,104 @@
 
 ![Build master](https://img.shields.io/travis/OpenST/organization-contracts/master.svg?label=build%20master&style=flat)
 ![Build develop](https://img.shields.io/travis/OpenST/organization-contracts/develop.svg?label=build%20develop&style=flat)
-![npm version](https://img.shields.io/npm/v/@openstfoundation/organization-contracts.svg?style=flat)
 [![Discuss on Discourse](https://img.shields.io/discourse/https/discuss.openst.org/topics.svg?style=flat)][discourse]
-[![Chat on Gitter](https://img.shields.io/gitter/room/OpenSTFoundation/SimpleToken.svg?style=flat)][gitter]
+[![Chat on Gitter](https://img.shields.io/gitter/room/ostdotcom/OST-Platform.svg?style=flat)][gitter]
 
-Organization contracts manage application permissions.
+An organization administers other contracts. For an individual contract or a system of contracts, an organization enables limiting execution of certain functions to the organization or other accounts it authorizes (e.g., "workers").
 
-## Instructions
+## Getting Started
 
-### Installation
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
 
-> Please add
+### Prerequisites
 
-### Usage
+Project requires [node] and [npm] to be installed.
 
-> Please add
+### Cloning
 
-## Related Work
-
-[mosaic-contracts] and [openst-contracts] use this package to manage access control.
+```bash
+git clone git@github.com:OpenST/organization-contracts.git
+```
 
-## Contributing
+### Installing
 
-### Set-up
+Install npm packages:
 
 ```bash
-git clone git@github.com:OpenST/organization-contracts.git
 cd organization-contracts
 npm install
+```
+## Usage
+
+The contracts and interfaces in this repository are primarily intended for use in other OpenST projects. Please see [Related Work].
+
+### Compiling the contracts
+
+The following npm script compiles updated contracts from the last call using [truffle]:
```

Comment:
"from the last call"?

Reply:
"from the last call" --> "since the last compile" (will change!)